scripts/release/mirror.js

```diff
@@ -0,0 +1,361 @@
+    newData.notes = updateNotes(
+      sourceData.notes,
+      notesRepl[0],
+      notesRepl[1],
+      (v) => getMatchingBrowserVersion(targetBrowser, v),
+    );
+  }
+
+  return newData;
+};
+
+/**
+ * @param {SupportStatement} sourceData
+ * @returns {SupportStatement}
+ */
+const bumpEdge = (sourceData) => {
+  if (
+    typeof sourceData.version_removed === 'string' &&
+    compareVersions.compare(sourceData.version_removed, '79', '<=')
+  ) {
+    // If this feature was removed before Chrome 79, it's not present in Chromium Edge
+    return { version_added: false };
+  }
+
+  return bumpGeneric(sourceData, 'edge', [/Chrome/g, 'Edge']);
+};
+
+/**
+ * @param {SupportStatement} sourceData
+ * @returns {SupportStatement}
+ */
+const bumpWebView = (sourceData) => {
+  let newData = copyStatement(sourceData);
+
+  const createWebViewRange = (version) => {
+    if (Number(version) <= 18) {
+      return '1';
+    } else if (Number(version) > 18 && Number(version) < 30) {
+      return '≤37';
+    } else if (Number(version) >= 30 && Number(version) < 33) {
+      return '4.4';
+    } else if (Number(version) >= 33 && Number(version) < 37) {
+      return '4.4.3';
+    } else {
+      return version;
+    }
+  };
+
+  if (typeof sourceData.version_added === 'string') {
+    newData.version_added = createWebViewRange(sourceData.version_added);
+  }
+
+  if (
+    sourceData.version_removed &&
+    typeof sourceData.version_removed === 'string'
+  ) {
+    newData.version_removed = createWebViewRange(sourceData.version_removed);
+  }
+
+  if (sourceData.notes) {
+    newData.notes = updateNotes(
+      sourceData.notes,
+      /Chrome/g,
+      'WebView',
+      createWebViewRange,
+    );
+  }
+
+  return newData;
+};
+
+/**
+ * @param {SupportStatement} data
+ * @param {string} destination
+ */
+export const bumpVersion = (sourceData, destination) => {
+  let newData = null;
+
+  if (sourceData == null) {
+    return null;
+  }
+
+  if (Array.isArray(sourceData)) {
+    return combineStatements(
+      ...sourceData.map((data) => bumpVersion(data, destination)),
+    );
+  }
+
+  if (destination === 'edge') {
+    newData = bumpEdge(sourceData);
+  } else if (destination === 'webview_android') {
+    newData = bumpWebView(sourceData);
+  } else {
+    let notesRepl;
+    if (destination.includes('opera')) {
+      notesRepl = [/Chrome/g, 'Opera'];
+    } else if (destination === 'samsunginternet_android') {
+      notesRepl = [/Chrome/g, 'Samsung Internet'];
+    }
+
+    newData = bumpGeneric(sourceData, destination, notesRepl);
+  }
+
+  if (!browsers[destination].accepts_flags && newData.flags) {
+    // Remove flag data if the target browser doesn't accept flags
+    return { version_added: false };
+  }
+
+  return newData;
+};
+
+const bumpData = (destination, data) => {
+  const upstream = browsers[destination].upstream;
+  return bumpVersion(data[upstream], destination);
+};
+
+export default bumpData;
```

Comment:
Since this is the main entry point, can you add tests for it? In particular the `combineStatements` simplification seems important to test.